src/app/components/Delegations/Delegation.tsx

```diff
@@ -162,6 +164,11 @@ export const Delegation: React.FC<DelegationProps> = ({
             <Tooltip id={`tooltip-${stakingTxHash}`} className="tooltip-wrap" />
           </div>
         </div>
+        {isApiNormal && !isGeoBlocked && (
```

Comment:
Wouldn't a staker only see this table if they connect their wallet, meaning that they are not geo-blocked?

Reply:
Sure